assets/js/dependencies/async_loader.js

```diff
@@ -256,3 +274,17 @@ _rStudioRequest=function(url,method,shared_secret,port,params){
     }
   });
 };
+
+// Read a page's GET URL variables and return them as an associative array.
+getUrlVars = function()
```

Comment:
From what I see both uses `window.location.href` so maybe a bug in `urlParam` ?
Better to fix it instead of creating a new function